lib/Backends/NNPI/tests/NNPIOperatorTest.cpp

```diff
@@ -278,6 +275,7 @@ struct BlacklistInitializer {
             {"ChannelwiseQuantizedConv3D/0", TestBlacklist::AnyDeviceAnyEngine},
             {"ChannelwiseQuantizedConv2D_NonZero_FloatBias/0",
              TestBlacklist::AnyDeviceSWEngine},
+            {"BatchBoxCox_Float16/0", TestBlacklist::AnyDeviceAnyEngine},
```

Comment:
Is this a temporary regression?

Reply:
Looks like this is a leftover of a test that does not exists anymore. I will remove it.